src/java/picard/sam/MergeSamFiles.java

```diff
@@ -83,6 +90,14 @@
     @Option(doc = "Comment(s) to include in the merged output file's header.", optional = true, shortName = "CO")
     public List<String> COMMENT = new ArrayList<String>();
 
+    @Option(shortName = "INTERVALS", doc = "An interval list file that contains the locations of the positions to merge. "+
+            "Assume bam are sorted and indexed. "+
+            "Reads should not overlap two intervals. "+
```

Comment:
this comment is unneed, what I meant is that the results of multiple runs of this program may contain the same read even if the regions that were used are disjoint, therefore the results should not be combined downstream without careful validation.